backend/src/main/java/gov/cdc/usds/simplereport/api/model/useraccountcreation/ActivateSecurityKeyRequest.java

```diff
@@ -0,0 +1,15 @@
+package gov.cdc.usds.simplereport.api.model.useraccountcreation;
+
+import javax.validation.constraints.Size;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public class ActivateSecurityKeyRequest {
+  @Size(max = RequestConstants.LARGE_REQUEST_STRING_LIMIT)
```

Comment:
I think these should be `@NotNull`, too.  Must have been in the last PR, but `EnrollMfaRequest` should also be `@NotNull`, right?

Reply:
Good catch, pretty much all of these request formatters should be `@NotNull` 😄  Updating now.